### PR TITLE
[AIFlow] Initialize database store and scheduler in forked process

### PR DESCRIPTION
**Pull request by Sxnan**
## What is the purpose of the change
Directly pass the scheduler and database store to the forked process can cause grpc to hang. Therefore, we initialize the database store and scheduler in the forked process.

## Brief change log

- Initialize database store and scheduler in forked process

## Verifying this change

This change added tests.

### CODE REVIEW

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py [hunk 1]
```diff
@@ -41,30 +37,35 @@ def __init__(self, namespace, event_type, key, sender):
...
 
 
-class WorkflowEventProcessor(Process):
-    def __init__(self, event_queue: Queue, store: AbstractStore, scheduler: Scheduler,
+class Poison(object):
+    pass
+
+
+class WorkflowEventProcessor:
+    def __init__(self, event_queue, store: AbstractStore, scheduler: Scheduler,
```
Comment (bgeng777) [inline flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:45]: The name of the first arg should be `conn` or sth else.

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_manager.py [hunk 2]
```diff
@@ -15,58 +15,78 @@
...
+        self.event_watcher = WorkflowEventWatcher(self.conn)
         self.listen_event_handler = None
-        self.event_processor = WorkflowEventProcessor(self.event_queue, store, scheduler)
+        self.event_processor_process = None
 
     def start(self):
         logging.info("WorkflowEventManager start listening event")
         self.listen_event_handler = self.notification_client.start_listen_events(self.event_watcher)
-        self.event_processor.start()
+        self.event_processor_process = Process(target=self._start_workflow_event_processor_process)
```
Comment (bgeng777) [inline flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_manager.py:68]: Is there any concern that initializing the Process() in start() method?

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py (hunk 1 above)
Comment (Sxnan) [reply flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py]: fixed in ae568b0

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_manager.py (hunk 2 above)
Comment (Sxnan) [reply flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_manager.py]: fixed in ae568b0

**Review by bgeng777 — APPROVED** [review]
LGTM

**Pull request merged**